Applications/Hermes/hermesflaskserver.py

```diff
@@ -8,6 +9,11 @@
 from flask import Flask
 from flask import request
 
+# Add Faraday library to the Python path.
+sys.path.append(os.path.join(os.path.dirname(__file__), '../..'))
```

Comment:
I get another import error when running this script:

![image](https://cloud.githubusercontent.com/assets/339144/25066973/3e7379e4-21ea-11e7-84e9-1e368cbf3a8c.png)